app/src/androidTest/java/space/taran/arknavigator/ResourceMetaInstrumentedTest.kt

```diff
@@ -0,0 +1,40 @@
+package space.taran.arknavigator
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.ActivityTestRule
+import org.junit.Assert.*
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import space.taran.arknavigator.mvp.model.repo.index.computeId
+import space.taran.arknavigator.ui.activity.MainActivity
+import java.io.File
+import java.io.FileOutputStream
+import java.io.InputStream
+
+
+@RunWith(AndroidJUnit4::class)
+class ResourceMetaTest {
+    @get:Rule
+    val mainActivityRule = ActivityTestRule(MainActivity::class.java)
+    @Test
+    fun crc32_iscorrect() {
```

Comment:
Looks good, but could you show where in the log this test is being run?
Can't find it in https://github.com/sisco0/ARK-Navigator/runs/4756014044

Reply:
This is not run under the CI as instrumented tests are required, we could work on running _Instrumented Tests_ remotely under the _CI_ but we should ensure that our environment uses an approach such as the one in https://github.com/marketplace/actions/android-emulator-runner. Hopefully, we could a new workflow for this or modify our already present one. What do you think?